packages/frontend/src/components/userActious/gitcoin/CheckingGitcoinScore.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components'
+import { FormRow, FormWrapper } from '../../form'
+import { TransactionStepper } from '@/components/stepper/Stepper'
+import { Colors } from '@/styles/colors'
+import { ClockIcon } from '@/components/icons'
+import { Button } from '@/components/buttons'
+
+const gitcoinScoreSteps = [
+  {
+    default: {
+      name: `Send request`,
```

Comment:
I wonder if it shouldn't be `Sending request` and "Sign the message" should be changed to `Signing the message`